src/command_modules/azure-cli-keyvault/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 unreleased
 ++++++++++++++++++++
 
+* Change `az keyvault certificate download` to better reflect the encoding options (PEM and DER).
```

Comment:
Since these options have changed, this is a breaking change, so please label it BC.